gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -128,6 +131,15 @@
   public static final DeregisterFileDeleteMethod DEFAULT_DEREGISTER_DELETE_METHOD =
       DeregisterFileDeleteMethod.NO_DELETE;
 
+  /**
+   * K-V pair to specify if {@link IMetaStoreClient }'s filtering method {@link listPartitionsByFilter} is not enough
+   * for filtering out specific partitions.
+   * For example, if you specify "Path" as the filter type and "Hourly" as the policy,
+   * partitions with Path containing '/Hourly/' will be kept.
+   */
+  public static final String HIVE_PARTITION_EXTENDED_FILTER_TYPE = HiveDatasetFinder.HIVE_DATASET_PREFIX + ".extendedFilterType";
+  public static final String HIVE_PARTITION_PATH_FILTER_REGEX = HiveDatasetFinder.HIVE_DATASET_PREFIX + ".pathFilterRegex";
```

Comment:
In general, statics should be in the scope where they are used. `HIVE_PARTITION_EXTENDED_FILTER_TYPE` is fine here because it is used for all Hive setups. However, `HIVE_PARTITION_EXTENDED_FILTER_TYPE` is only used for one particular filter, so you should put it in that filter.